hack/make-rules/test-e2e-node.sh

```diff
@@ -149,7 +150,7 @@ if [ "${remote}" = true ] ; then
     --image-project="${image_project}" --instance-name-prefix="${instance_prefix}" \
     --delete-instances="${delete_instances}" --test_args="${test_args}" --instance-metadata="${metadata}" \
     --image-config-file="${image_config_file}" --system-spec-name="${system_spec_name}" \
-    --extra-envs="${extra_envs}" --test-suite="${test_suite}" \
+    --preemptible-instances="${preemptible_instances}" -extra-envs="${extra_envs}" --test-suite="${test_suite}" \
```

Comment:
One more "-" seems necessary for `-extra-envs`

Reply:
Nice catch, thanks!